mne/time_frequency/_stockwell.py

```diff
@@ -156,6 +156,14 @@ def _induced_power_stockwell(data, sfreq, fmin, fmax, n_fft=None, width=1.0,
         (2006). S-transform time-frequency analysis of P300 reveals deficits in
         individuals diagnosed with alcoholism.
         Clinical Neurophysiology 117 2128--2143
+
+    See Also
+    --------
+    mne.time_frequency.tfr_stockwell
+    mne.time_frequency.tfr_multitaper
+    mne.time_frequency.tfr_array_multitaper
+    mne.time_frequency.tfr_morlet
+    mne.time_frequency.tfr_array_morlet
```

Comment:
FYI you don't need to put the full path for functions in the same module, e.g. `tfr_stockwell` should work as well as `mne.time_frequency.tfr_stockwell` here